plugins/smart_proxy_remote_execution_ssh/ruby-smart-proxy-remote-execution-ssh.postinst

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+# postinst script for ruby-smart_proxy_remote_execution_ssh
+#
+# see: dh_installdeb(1)
+
+set -e
+
+USERNAME="foreman-proxy"
+GROUPNAME="foreman-proxy"
+FOREMAN_PROXY_HOME="/usr/share/foreman-proxy"
+VARDIR="/var/lib/foreman-proxy"
+
+case "$1" in
+  configure)
+    if [ -d $VARDIR ] ; then
+      mkdir -p $VARDIR
+      chown $USERNAME:$GROUPNAME $VARDIR
+    fi
+
+    if [ -d $FOREMAN_PROXY_HOME/.ssh ] && [ ! -L $FOREMAN_PROXY_HOME/.ssh ] ; then
+      mv $FOREMAN_PROXY_HOME/.ssh $VARDIR/ssh
```

Comment:
this will move the keys to `$VARDIR/ssh/.ssh` is that correct?

Reply:
If the directory doesn't exist then it will rename `.ssh` to `ssh`. By default this is the case. If it did exist, then it would indeed end up in a subdirectory. Could be a feature not to overwrite files :)

Reply:
but the dir does exist, you created it.

Reply:
The parent dir exists. It might be cleaner to rely on the main foreman-proxy package to manage `/var/lib/foreman-proxy` but this allows us to ship the fix async. However, given we'll also need an installer update I'm debating if that's the best way forward.

```
$ mkdir /tmp/.ssh
$ mkdir /tmp/bla
$ mv /tmp/.ssh /tmp/bla/ssh
$ ls -l /tmp/bla
total 0
drwxrwxr-x. 2 ekohl ekohl 40 23 aug 16:57 ssh
```

Reply:
oh right, you only create up to VARDIR, sry.